src/Microsoft.Azure.WebJobs.Host/Bindings/Path/BindingParameterResolver.cs

```diff
@@ -74,28 +75,36 @@ protected string GetFormatOrNull(string value)
             return null;
         }
 
-        // This is an alias for 'sys.randguid'
-        private class RandGuidResolver : BindingParameterResolver
+        // This is an alias for 'sys.guid'
+        private class GuidResolver : BindingParameterResolver
```

Comment:
But we want to deprecate this altogether!   I never want somebody to write {guid}; only {sys.guid} . This is adding a codepath that we never want somebody to actually take. 



---
In reply to: [118123998](https://github.com/Azure/azure-webjobs-sdk/pull/1157#discussion_r118123998) [](ancestors = 118123998)